cfgov/unprocessed/js/molecules/Autocomplete.js

```diff
@@ -121,20 +140,56 @@ function Autocomplete( element, opts ) {
       inputCoords.left ) + 'px';
   }
 
+  /**
+   * Toggles the error state and message that's shown when the
+   * max search term length is hit
+   */
+  function _toggleMaxLengthError() {
+    _maxLengthExceeded = _searchTerm.length >= _settings.maxChars ? true :
+      false;
+    if ( _maxLengthExceeded ) {
+      _input.classList.add( ERROR_CLASS );
+
+      /* If you type fast enough, search results from a letter or two back can
+         cover up the max length error, so we'll wait a bit before hiding */
+      setTimeout( _hide, _settings.delay );
+      if ( _settings.errorMessage ) {
+        _settings.errorMessage.classList.remove( HIDDEN_CLASS );
+      }
+    } else {
+      _input.classList.remove( ERROR_CLASS );
+      if ( _settings.errorMessage ) {
+        _settings.errorMessage.classList.add( HIDDEN_CLASS );
+      }
+    }
+    _this.dispatchEvent( 'maxCharacterChange', {
+      maxLengthExceeded: _maxLengthExceeded
+    } );
+  }
+
   /**
    * Binds input, blur, and keydown events on _input element and
    * resize event on window.
    */
   function _bindEvents() {
     _input.addEventListener( 'input', function() {
```

Comment:
I'd probably move the event handler function to it's own named function, like is done with `handleMaxCharacters`. It should get an `_` prefix on the name as it isn't exported from `Autocomplete`.

Reply:
Done in 8b4c07f—the `input` event listener now calls `_handleInput` instead of an anonymous function.

Reply:
Hmm relevant to the `_this` question, you could try 

```
const _handleInputBinded = _handleInput.bind( this );
_input.addEventListener( 'input', _handleInputBinded );
```
… and the value of `this` should be correct inside `_toggleMaxLengthError `. You'd need to change `_searchTerm = this.value;` to get the value through the event parameter though.

Reply:
See https://github.com/cfpb/consumerfinance.gov/pull/6537#discussion_r680220429. Since we're keeping `const _instance = this`, I left this as is